app/views/shared/processed_and_deleted/_index.html.slim

```diff
@@ -8,7 +8,7 @@
       table *{class: "#{state}-applications"}
         thead
           tr
-            th =t('processed_applications.table_header.applicant')
+            = table_header
```

Comment:
As for using the helper:

```ruby
ReferenceHelper.table_header
```

it throws an error using it that way also all the other helpers are used without specifying the module name.